### PR TITLE
Add some notes on Archivematica concepts

### PR DESCRIPTION
**Pull request by alexwlchan**

### CODE REVIEW

File: docs/conceptual-bits.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Conceptual bits that are useful for debugging Archivematica
+
+This page has some notes on conceptual bits that were useful as I was setting up/debugging our Archivematica deployment.
+Some of it is specific to Wellcome, some of it is general Archivematica knowledge.
+
+## Microservices/tasks/jobs
```
Comment (jamesgorrie) [inline docs/conceptual-bits.md]: Might make this heading:
```suggestion
## Microservices, tasks and jobs
```

As I thought you might be reffering to the same thing.

File: docs/conceptual-bits.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+The top level unit is the *microservice*, for example `Create SIP from Transfer`.
+
+Each microservice runs a number of *jobs*.
+Each job is doing a different action – for example, `Check transfer directory for objects` or `Load options to create SIPs`.
+
+Each job may spawn one or more *tasks*, which are the Python scripts that run under the hood.
+You can see the tasks by clicking the gear icon.
+Often tasks run on a per-file basis: if there are 100 files in a transfer package and you need to perform an action on each file, there would be 100 tasks.
+
+Microservices contain jobs, jobs spawn tasks:
```
Comment (jamesgorrie) [inline docs/conceptual-bits.md]: 👍 nice and clear.

File: docs/conceptual-bits.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+![](images/microservices.svg)
+
+Sometimes actions get stuck and need to be restarted; the only way I know how to do this is to restart the Archivematica containers (more on that below).
```
Comment (jamesgorrie) [inline docs/conceptual-bits.md]: Could you link to a heading

**Comment by alexwlchan** [comment]
Part of https://github.com/wellcomecollection/platform/issues/4342

**Pull request merged**